.isort.cfg

```diff
@@ -2,3 +2,6 @@
 line_length=120
 multi_line_output=0
 lines_after_imports=2
+sections=FUTURE,STDLIB,THIRDPARTY,INDICO,FIRSTPARTY,LOCALFOLDER
+known_third_party=flask_multipass,flask_pluginengine
+known_indico=indico
```

Comment:
This shouldn't be needed as long as flask-multipass is installed in the environment when running isort - and the first/last line and flask_pluginengine in the second line wouldn't be needed here in any case